### PR TITLE
small fixes to support Tableau

### PR DESCRIPTION
**Pull request by fokot**
Less errors but not working https://coralogix.atlassian.net/browse/DEV-2119

### CODE REVIEW

File: USAGE.md [hunk 1]
```diff
@@ -59,19 +59,21 @@ We will show you how to do it for three popular database tools. Procedure with o
    | Windows | C:\Program Files\Tableau\Drivers |
    | Mac     | ~/Library/Tableau/Drivers        |
    | Linux   | /opt/tableau/tableau_driver/jdbc |
-3. Create file `coralogix.props` and add there line with your `apiKey`.
+3. Copy [coralogix.tdc](coralogix.tdc) file in `~/Documents/My Tableau Repository/Datasources`
```
Comment (iravid) [inline USAGE.md]:
```suggestion
3. Copy the [coralogix.tdc](coralogix.tdc) file to `~/Documents/My Tableau Repository/Datasources`
```

File: USAGE.md [hunk 2]
```diff
@@ -59,19 +59,21 @@ We will show you how to do it for three popular database tools. Procedure with o
...
+   For more details consult [Tableau documentation](https://kb.tableau.com/articles/howto/using-a-tdc-file-with-tableau-server)
+4. Create file `coralogix.properties` and add there line with your `apiKey`.
```
Comment (iravid) [inline USAGE.md]:
```suggestion
4. Create a file named `coralogix.properties` and add an entry with your `apiKey`.
```

**Review by iravid — APPROVED** [review]

File: USAGE.md (hunk 1 above)
Comment (fokot) [reply USAGE.md]: Fixed

File: USAGE.md (hunk 2 above)
Comment (fokot) [reply USAGE.md]: Fixed

**Pull request merged**